libs/test/SalesforceSDKTest/AndroidManifest.xml

```diff
@@ -27,4 +27,6 @@
 
     <uses-sdk android:minSdkVersion="17"
         android:targetSdkVersion="21" />
+
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Need this to create a file for the upload test (we didn't have a file upload test).